src/js/control-bar/playback-rate-menu-button.js

```diff
@@ -98,7 +98,7 @@ PlaybackRateMenuButton.prototype.updateVisibility = function(){
  */
 PlaybackRateMenuButton.prototype.updateLabel = function(){
   if (this.playbackRateSupported()) {
-    this.labelEl_.innerHTML = this.player().playbackRate() + 'x';
+    this.labelEl_.innerHTML = `${this.player().playbackRate()}x`;
```

Comment:
I went back and forth on this one, but I think we should err on the side of consistency. Once we get used to expecting string interpolation, my hope is that this will look normal.